onnxmltools/utils/main.py

```diff
@@ -71,7 +72,7 @@ def save_text(model, file_path):
     if model is None or not isinstance(model, onnx_proto.ModelProto):
         raise ValueError("Model is not a valid ONNX model.")
     with open(file_path, "w") as f:
-        f.write(str(model))
+        f.write(json.dumps(str(model)))
```

Comment:
What's the problem of the existing text output?

Reply:
save_text does not generate json object.